src/shared/components/Button/Button.tsx

```diff
@@ -8,7 +8,7 @@ import { TextVariant } from '../Text'
 export type ButtonProps = {
   icon?: React.ReactNode
   iconPlacement?: IconPlacement
-  children: string
+  children?: string
```

Comment:
I don't think you should edit this. You should use `IconButton` instead of regular `Button` and pass outline as children like this 
```js
// ChannelView.tsx
              <SearchButton
                onClick={() => {
                  setIsSearchingInputOpen(true)
                  searchInputRef.current?.focus()
                }}
                variant="tertiary"
              >
                <SvgOutlineSearch2 />
              </SearchButton>
```
```js
//ChannelView.style.ts

export const SearchButton = styled(IconButton)`
  position: absolute;
`
```